crawler/file_processing.py

```diff
@@ -633,92 +803,6 @@ def get_errored_ids(error: Dict[str, Dict[str, str]]) -> str:
             logger.exception(e)
             return []
 
```

Comment:
Move functions insert_samples_from_docs_into_mlwh and insert_plates_and_wells_from_docs_into_dart  again after this function so it won't appear as a change